app/javascript/components/admin/server_rooms/ServerRoomRow.jsx

```diff
@@ -13,25 +13,37 @@ import useRoomStatus from '../../../hooks/queries/rooms/useRoomStatus';
 import { useAuth } from '../../../contexts/auth/AuthProvider';
 
 export default function ServerRoomRow({ room }) {
-  const { friendly_id: friendlyId } = room;
+  const {
+    friendly_id: friendlyId, name, owner, last_session: lastSession, status, participants,
+  } = room;
   const mutationWrapper = (args) => useDeleteServerRoom({ friendlyId, ...args });
   const { handleStartMeeting } = useStartMeeting(friendlyId);
   const currentUser = useAuth();
   // TODO - samuel: useRoomStatus will not work if room has an access code. Will need to add bypass in MeetingController
   const { refetch } = useRoomStatus(room.friendly_id, currentUser.name);
 
+  const renderLastSession = () => {
+    if (lastSession == null) {
+      return <span> No meeting yet. </span>;
+    }
+    if (status === 'Active') {
+      return <span> Started: {lastSession} </span>;
+    }
+    return <span> Ended: {lastSession} </span>;
+  };
+
   return (
     <tr className="align-middle text-muted border border-2">
       <td className="border-end-0">
         <Stack>
-          <span className="text-dark fw-bold"> {room.name} </span>
-          <span> Ended: </span>
+          <span className="text-dark fw-bold"> {name} </span>
+          {renderLastSession()}
```

Comment:
I would keep the span here and just return text instead `<span> ${ lastSession } </span>`

Reply:
![image](https://user-images.githubusercontent.com/43917914/179251352-fee19aec-72b8-4b3d-a08a-9cfa0dd6ab87.png)